### PR TITLE
[WIP] Core components rename 

### PR DESCRIPTION
**Pull request by Igloczek**

### CODE REVIEW

File: core/components/ColorSelector.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import genericSelector from 'core/components/genericSelector'
+import genericSelector from 'core/components/GenericSelector'
```
Comment (pkarw) [inline core/components/ColorSelector.js]: Why not GenericSelector?

File: src/themes/catalog/components/core/Breadcrumbs.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
 </template>
 
 <script>
-import breadcrumbs from 'core/components/modal'
+import breadcrumbs from 'core/components/Modal'
```
Comment (pkarw) [inline src/themes/catalog/components/core/Breadcrumbs.vue]: Why not Breadcrumbs?

File: src/themes/catalog/components/core/Notification.vue [hunk 3]
```diff
@@ -12,7 +12,7 @@
 </template>
 
 <script>
-import notification from 'core/components/blocks/notification'
+import notification from 'core/components/blocks/Notification'
```
Comment (pkarw) [inline src/themes/catalog/components/core/Notification.vue]: Why not Notification?

File: src/themes/default/components/core/blocks/Wishlist/Wishlist.vue [hunk 4]
```diff
@@ -25,7 +25,7 @@
 </template>
 
 <script>
-import wishlist from 'core/components/blocks/Wishlist/wishlist'
+import wishlist from 'core/components/blocks/Wishlist/Wishlist'
```
Comment (pkarw) [inline src/themes/default/components/core/blocks/Wishlist/Wishlist.vue]: Should be Wishlist

File: src/themes/theme-starter/components/ProductTile.vue [hunk 5]
```diff
@@ -51,7 +51,7 @@
 </template>
 
 <script>
-import productTile from 'core/components/productTile'
+import productTile from 'core/components/ProductTile'
```
Comment (pkarw) [inline src/themes/theme-starter/components/ProductTile.vue]: Should be ProductTile

File: src/themes/default/components/core/blocks/Wishlist/RemoveButton.vue [hunk 6]
```diff
@@ -6,7 +6,7 @@
 </template>
 
 <script>
-import removeButton from 'core/components/blocks/Wishlist/removeButton'
+import removeButton from 'core/components/blocks/Wishlist/RemoveButton'
```
Comment (pkarw) [inline src/themes/default/components/core/blocks/Wishlist/RemoveButton.vue]: Should be RemoveButton

File: src/themes/catalog/components/core/blocks/Wishlist/Product.vue [hunk 7]
```diff
@@ -22,7 +22,7 @@
 </template>
 
 <script>
-import product from 'core/components/blocks/Wishlist/product'
+import product from 'core/components/blocks/Wishlist/Product'
```
Comment (pkarw) [inline src/themes/catalog/components/core/blocks/Wishlist/Product.vue]: Should be Product

File: src/themes/default/components/core/blocks/Wishlist/Product.vue [hunk 8]
```diff
@@ -36,7 +36,7 @@
 </template>
 
 <script>
-import product from 'core/components/blocks/Wishlist/product'
+import product from 'core/components/blocks/Wishlist/Product'
```
Comment (pkarw) [inline src/themes/default/components/core/blocks/Wishlist/Product.vue]: Should be Product

File: src/themes/default/components/core/blocks/Category/Sidebar.vue [hunk 9]
```diff
@@ -78,7 +78,7 @@
 </template>
 
 <script>
-import sidebar from 'core/components/blocks/Category/sidebar'
+import sidebar from 'core/components/blocks/Category/Sidebar'
```
Comment (pkarw) [inline src/themes/default/components/core/blocks/Category/Sidebar.vue]: Should be Sidebar

File: core/components/SizeSelector.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import genericSelector from 'core/components/genericSelector'
+import genericSelector from 'core/components/GenericSelector'
```
Comment (pkarw) [inline core/components/SizeSelector.js]: GenericSelector

File: src/themes/catalog/components/core/blocks/Product/ProductAttribute.vue [hunk 11]
```diff
@@ -6,7 +6,7 @@
 </template>
 
 <script>
-import productAttribute from 'core/components/productAttribute'
+import productAttribute from 'core/components/ProductAttribute'
```
Comment (pkarw) [inline src/themes/catalog/components/core/blocks/Product/ProductAttribute.vue]: ProductAttribute

File: src/themes/catalog/components/core/Breadcrumbs.vue (hunk 2 above)
Comment (Igloczek) [reply src/themes/catalog/components/core/Breadcrumbs.vue]: Good catch!

**Comment by pkarw** [comment]
Thanks, will be merged in soon

**Pull request merged**